src/main/java/org/icgc/argo/program_service/converter/ProgramConverter.java

```diff
@@ -57,6 +58,8 @@ public interface ProgramConverter {
   @Mapping(target = "dataCenterId", ignore = true)
   ProgramEntity programToProgramEntity(Program p);
 
+  DataCenterDTO dataCenterToDataCenterEntity(DataCenterEntity p);

```

Comment:
Similarly, a new converter for Data Centers, this is a separate entity which should keep its own code separate

Reply:
Moved the mapping to DataCenterConvertor